packages/react-charting/src/components/VerticalBarChart/VerticalBarChart.base.tsx

```diff
@@ -699,7 +699,7 @@ export class VerticalBarChartBase extends React.Component<IVerticalBarChartProps
       this._classNames = getClassNames(this.props.styles!, {
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30306&commitId=ffaed3b0d30ceed09d709be0e0e64ea89917762e&env=prod&clientType=FLUENTUI)


  <details><summary><b>Combobox Converged</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Combobox Converged.Open.active option.chromium.png|1100| Changed|
|Combobox Converged.Disabled option.active option.chromium.png|1100| Changed|
|Combobox Converged.Open with inlinePopup.active option.chromium.png|1100| Changed|
|Combobox Converged.With selection.active option.chromium.png|1100| Changed|
|Combobox Converged.Option with long content.active option.chromium.png|1578| Changed|
|Combobox Converged.Open with grouped options.active option.chromium.png|1100| Changed|

</details>


  <details><summary><b>Dropdown Converged</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dropdown Converged.Open with inlinePopup.active option.chromium.png|502| Changed|
|Dropdown Converged.Disabled option.active option.chromium.png|502| Changed|
|Dropdown Converged.Option with long content.active option.chromium.png|502| Changed|
|Dropdown Converged.With selection.active option.chromium.png|502| Changed|
|Dropdown Converged.Open.active option.chromium.png|502| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>